BUILD.gn

```diff
@@ -887,6 +887,22 @@ group("licenses") {
   ]
 }
 
+copy("electron_mksnapshot") {
+  if (is_win) {
```

Comment:
Not sure if its needed but I have seen elsewhere in BUILD.gn files a declaration of the variable outside an if -- also GN supports else, eg:
```sources = []
if (is_win) {
  sources += "mksnapshot.exe",
} else {
  sources += "mksnapshot",
}```